packages/aws-cdk/bin/cdk.ts

```diff
@@ -56,6 +57,7 @@ async function parseCommandLineArguments() {
       .option('bootstrap-kms-key-id', { type: 'string', desc: 'AWS KMS master key ID used for the SSE-KMS encryption', default: undefined })
       .option('tags', { type: 'array', alias: 't', desc: 'Tags to add for the stack (KEY=VALUE)', nargs: 1, requiresArg: true, default: [] })
       .option('execute', {type: 'boolean', desc: 'Whether to execute ChangeSet (--no-execute will NOT execute the ChangeSet)', default: true})
+      .option('new-bootstrap', { type: 'boolean', desc: 'Whether to use the experimental new enhanced bootstrapping needed for the "CI/CD for CDK apps" story. Default is the classic bootstrapping', default: false })
```

Comment:
It needs to be an option, but I wonder if we should hide it more. Either disclaim it more here ("will change in the future, experimental, don't use etc tec") or maybe make it an environment variable that toggles this?

Reply:
+1 for environment variable

Reply:
Changed to an env variable.